meson.build

```diff
@@ -0,0 +1,18 @@
+project('UnitTest++', 'cpp',
+    version : '2.0.0',
+    license : 'MIT')
+
+unittest_cpp_src = []
+
+subdir('UnitTest++')
+
+unittest_cpp_lib = static_library('unittest-cpp', 
+    unittest_cpp_src,
+    include_directories : '.')
+
+unittest_cpp_dep = declare_dependency(
+    version : meson.project_version(),
+    link_with : unittest_cpp_lib,
+    include_directories : '.')
+
+# subdir('tests')
```

Comment:
Same question.

Reply:
```console
Found runner: /Users/mike/Library/Python/3.8/bin/ninja
ninja: Entering directory `build'
[31/46] Compiling C++ object tests/TestUnitTest++.p/TestTestMacros.cpp.o
FAILED: tests/TestUnitTest++.p/TestTestMacros.cpp.o 
c++ -Itests/TestUnitTest++.p -Itests -I../tests -I. -I.. -Xclang -fcolor-diagnostics -pipe -Wall -Winvalid-pch -Wnon-virtual-dtor -g -MD -MQ tests/TestUnitTest++.p/TestTestMacros.cpp.o -MF tests/TestUnitTest++.p/TestTestMacros.cpp.o.d -o tests/TestUnitTest++.p/TestTestMacros.cpp.o -c ../tests/TestTestMacros.cpp
../tests/TestTestMacros.cpp:167:27: error: expected ';' at end of declaration list
      ~FixtureDtorThrows() _NOEXCEPT_OP(false) {
                          ^
                          ;
1 error generated.
[44/46] Compiling C++ object tests/TestUnitTest++.p/TestExceptions.cpp.o
ninja: build stopped: subcommand failed.
```

Reply:
Could you please add a comment above the commented out line explaining this? It's really nice for any future contributors to know why tests are disabled. Also if it's only a single test that is broken can we disabled just this one and enable other tests? Tests help a lot to verify that the submission works and the future revisions of the wrap still work.

Reply:
Would this work?

```
#
# Just a warning before you remove the hash symbol from "subdir('tests')",
# it generates the following error message.
#
# tests/TestTestMacros.cpp:167:27: error: expected ';' at end of declaration list
#      ~FixtureDtorThrows() _NOEXCEPT_OP(false) {
#                          ^
#                          ;
#
```

Reply:
Ok apparently commented out the wrong file. The problem came from `TestTestMacros.cpp` and by disabling the file I was able to run the tests.